daml-lf/spec/daml-lf-1.rst

```diff
@@ -1713,6 +1770,12 @@ name* construct as follows:
 * The *fully resolved name* of a choice ``Ch`` of a template
   definition ``'tpl' (x : T) ↦ { …, 'choices' { …, 'choice' ChKind Ch
   … ↦ …, … } }`` defined in the module ``Mod`` is ``Mod.T.Ch``.
+* The *fully resolved name* of an inherited choice ``Ch`` of a template
+  definition ``'tpl' (x : T) ↦ { …, 'implements' {  …, 'choices' { …, Ch,
+  … } } }`` defined in the module ``Mod`` is ``Mod.T.Ch``.
+* The *fully resolved name* of a choice ``Ch`` of an interface
+  definition ``'interface' (x : I) ↦ { …, 'choices' { …, 'choice' ChKind Ch
+  … ↦ …, … } }`` defined in the module ``Mod`` is ``Mod.I.Ch``.
```

Comment:
It maybe safer to include the method name.

Reply:
That makes sense (e.g. don't want a collision between an interface choice and an interface method). I'll update the name collision checker separately.